image_server/server_setup.md

```diff
@@ -78,7 +78,6 @@ If setting up in a browser, go to the CloudFormation service in the AWS console
 
 
 2. Specify parameters - some hints are:
-   * Bucket - S3 bucket where data are stored. Disasters data are generally stored in `hyp3-nasa-disasters`
```

Comment:
Is there a reason to delete this guidance rather than update it? Perhaps it would be worth using this hint to clarify that prefixes aren't included, nor is the `s3://`.

Reply:
The cloudformation stack doesn't have a `Bucket` parameter anymore. It did at one point, but we gradually interacted with more and more buckets that we hard-coded that list at https://github.com/ASFHyP3/gis-services/blob/develop/image_server/cloudformation.yml#L166